### PR TITLE
update test results summary after BLS 0.10.1 update

### PR DESCRIPTION
**Pull request by tersec**
Intentionally isolating this in its own PR.

The removed tests are legitimately removed/skipped for now, but random PRs ideally shouldn't have to be updating this -- so, this way, make the awareness/consciousness clear.

### CODE REVIEW

File: AllTests-minimal.md [hunk 1]
```diff
@@ -205,11 +198,10 @@ OK: 9/9 Fail: 0/9 Skip: 0/9
 OK: 1/1 Fail: 0/1 Skip: 0/1
 ## [Unit - Spec - Block processing] Attestations  [Preset: minimal]
 ```diff
-+ Empty aggregation bit                                                                      OK
```
Comment (mratsim) [inline AllTests-minimal.md]: I think this one should appear in SKIPPED

File: FixtureAll-mainnet.md [hunk 2]
```diff
@@ -68,11 +68,9 @@ OK: 20/20 Fail: 0/20 Skip: 0/20
 ```diff
 + [Invalid] att1_bad_extra_index                                                             OK
 + [Invalid] att1_bad_replaced_index                                                          OK
-+ [Invalid] att1_duplicate_index_double_signed                                               OK
```
Comment (mratsim) [inline FixtureAll-mainnet.md]: and this

File: FixtureAll-mainnet.md [hunk 3]
```diff
@@ -68,11 +68,9 @@ OK: 20/20 Fail: 0/20 Skip: 0/20
...
 + [Invalid] att1_duplicate_index_normal_signed                                               OK
 + [Invalid] att2_bad_extra_index                                                             OK
 + [Invalid] att2_bad_replaced_index                                                          OK
-+ [Invalid] att2_duplicate_index_double_signed                                               OK
```
Comment (mratsim) [inline FixtureAll-mainnet.md]: and this

File: FixtureAll-mainnet.md [hunk 4]
```diff
@@ -129,7 +127,6 @@ OK: 2/2 Fail: 0/2 Skip: 0/2
 OK: 9/9 Fail: 0/9 Skip: 0/9
 ## Official - Sanity - Blocks  [Preset: mainnet]
 ```diff
-+ [Invalid] expected_deposit_in_block                                                        OK
```
Comment (mratsim) [inline FixtureAll-mainnet.md]: and this

**Comment by arnetheduck** [comment]
shouldn't CI check that the MD is accurate, so that if someone forgets to update it, it fails?

File: FixtureAll-mainnet.md (hunk 4 above)
Comment (tersec) [reply FixtureAll-mainnet.md]: okay, well, that might be true, regarding all these "OK" statuses that should be "Skipped", but I just wanted to get `devel` internally consistent

File: FixtureAll-mainnet.md (hunk 3 above)
Comment (tersec) [reply FixtureAll-mainnet.md]: As with the others, this is just rendering consistent/reaching a fixpoint on the interaction between https://github.com/status-im/nim-beacon-chain/pull/794 and https://github.com/status-im/nim-beacon-chain/pull/780

**Pull request merged**